packages/api/src/external/commonwell/patient.ts

```diff
@@ -136,6 +136,33 @@ export async function create(
   }
 }
 
+export async function get(
+  patient: Patient,
+  facilityId: string
+): Promise<CommonwellPatient | undefined> {
+  const { debug } = Util.out(`CW get - M patientId ${patient.id}`);
```

Comment:
Seems to be using the deprecated Util.Out

Reply:
I was following the other functions on the same file, but updated them all to use the active one. 👍